### PR TITLE
Update for trigger renames for wpilibsuite/allwpilib#4210

### PR DESCRIPTION
**Pull request by sciencewhiz**

### CODE REVIEW

**Comment by Starlight220** [comment]
Note the semantics of `whileActiveContinuously`/`whileHeld` being replaced by either `whileTrue(RepeatCommand)` for most commands or `whileTrue(RunCommand)` if it's an `InstantCommand`.

**Comment by sciencewhiz** [comment]
> Note the semantics of `whileActiveContinuously`/`whileHeld` being replaced by either `whileTrue(RepeatCommand)` for most commands or `whileTrue(RunCommand)` if it's an `InstantCommand`.

That's a significant change to RobotBuilder to support different semantics based on what type of command it is.

**Comment by Starlight220** [comment]
> > Note the semantics of `whileActiveContinuously`/`whileHeld` being replaced by either `whileTrue(RepeatCommand)` for most commands or `whileTrue(RunCommand)` if it's an `InstantCommand`.
> 
> That's a significant change to RobotBuilder to support different semantics based on what type of command it is.

That conditional is the migration guide; it shouldn't be in code anywhere (unless you're planning on automatic migration, which is a different story).

File: src/test/java/robotbuilder/TestUtils.java [hunk 1]
```diff
@@ -177,11 +177,11 @@ public static RobotTree generateTestTree() {
         leftstick.add(armUpButton);
         RobotComponent autoButton = new RobotComponent("Autonomous Button", "Joystick Button", tree);
         rightstick.add(autoButton);
-        autoButton.getProperty("When to Run").setValueAndUpdate("whenPressed");
+        autoButton.getProperty("When to Run").setValueAndUpdate("onTrue");
         RobotComponent xbox = new RobotComponent("Xbox", "Xbox Controller", tree);
         oi.add(xbox);
         RobotComponent xboxButton = new RobotComponent("Arm Up Xbox Button", "Xbox Button", tree);
-        xboxButton.getProperty("When to Run").setValueAndUpdate("whenPressed");
+        xboxButton.getProperty("When to Run").setValueAndUpdate("whileTrue");
```
Comment (Starlight220) [inline src/test/java/robotbuilder/TestUtils.java:184]: This looks like a copypasta mistake

Comment (sciencewhiz) [reply src/test/java/robotbuilder/TestUtils.java:184]: that was intentional to get coverage of additional types

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**